core-java-modules/core-java-uuid/src/test/resources/logback.xml

```diff
@@ -2,8 +2,7 @@
 <configuration>
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
-            </pattern>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
```

Comment:
No reason to make this change as it is a whitespace-only change and not a functional change.